torch/testing/_internal/common_utils.py

```diff
@@ -1800,7 +1800,7 @@ def check_if_enable(test: unittest.TestCase):
             raise unittest.SkipTest(skip_msg)
 
     if TEST_SKIP_FAST:
-        if not getattr(test, test._testMethodName).__dict__.get('slow_test', False):
+        if hasattr(test, test._testMethodName) and not getattr(test, test._testMethodName).__dict__.get('slow_test', False):
```

Comment:
Could you move this check `hasattr(test, test._testMethodName)` to a separate if condition and raise a separate skip message for this?  Otherwise, it would be hard to tell them apart

Reply:
i don't think it makes sense to have a separate skip.  In my patch, I am not raising in the case that test doesn't have _testMethodName.

my operating theory is that this is just an invalid thing but i did not fix it properly, wondering if its useful to land to unblock CI or if it makes anything worse.

Reply:
I see. I read the condition wrong (the other way round).  If I understand this correctly now, if the attributed is not there, the test won't be skipped and it will run as usual.  That looks correct to me

Reply:
right, agreed about the first part (the condition and not skipping the test).  And i also don't understand the test setup well so i can't really explain the second part.. :/

idk if we want to land this or dig more.  I thought its unblocking CI but as I look at HUD now I don't see the periodic failure.  Is it just bc the test is no longer marked slow?

Reply:
Yup, the list of slow test is updated daily.  Any test taking longer than 60s is included, and those less than that is removed.  So it could be a bit of a flux for borderline ones